.github/workflows/pypi.yml

```diff
@@ -0,0 +1,46 @@
+# They are provided by a third-party and are governed by
```

Comment:
@domanchi Proposing that we add a github action for deploying to pypi. We would still use the bump-version script to bump the version and push the tag. Once the tag is pushed - the github action would do the actual deploy to pypi, replacing the upload-to-pypi script.

Reply:
what does this code comment mean? (what's `They`?)

Reply:
Ops this was generated by github when creating the template action. Looks like part of it got cut off. Will add the full generated comment.

Reply:
Schweet -- on new tag, then auto-upload sounds like a great addition.